include/mxnet/lib_api.h

```diff
@@ -1111,16 +1197,49 @@ extern "C" {
 
     // create a vector of tensors for inputs
     std::vector<MXTensor> inputs(num_in);
+    // create a vector for sparse inputs
+    std::vector<MXInSparse> in_sparse(num_in);
```

Comment:
This assumes we should allocate a sparse tensor for every input, before even checking if the input is sparse. Maybe theres a better way to do it